dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsolute.java

```diff
@@ -10,6 +10,7 @@
 import com.dotmarketing.portlets.templates.design.bean.TemplateLayout;
```

Comment:
This should not need to change because this task should only be run on an upgrade, and should be run before the contentlet has been converted to json